scripts/test_integration.py

```
@@ -0,0 +1,306 @@
+from unittest.mock import MagicMock, patch
+from io import StringIO
+
+# Add the project root to the path
+project_root = Path(__file__).parent.parent
+sys.path.insert(0, str(project_root))
+
+from sphinxcontrib.jsontable.directives import (
+    TableConverter, 
+    JsonTableDirective,
+    DEFAULT_MAX_ROWS
+)
```

Ruff (0.11.9)

Module level import not at top of file

(E402)